granulate_utils/containers/cri.py

```diff
@@ -38,9 +39,10 @@ class CriClient(ContainersClientInterface):
     def __init__(self) -> None:
         self._runtimes = {}
         for rt, path in RUNTIMES:
-            path = "unix://" + ns.resolve_host_root_links(path)
-            if self._is_cri_available(path):
-                self._runtimes[rt] = path
+            if os.path.exists(path):
```

Comment:
Not sure what better we can do about it.
It seems reasonable to put this check here, and prevent grpc from retrying connection to unavailable unix socket.